docs/whats-new/dotnet-aspire-9.2.md

```diff
@@ -0,0 +1,516 @@
+
+Each container app automatically gets its own **managed identity**, and Aspire now generates the necessary role assignment infrastructure for both default and per-reference roles. When targeting existing Azure resources, role assignments are scoped correctly using separate Bicep resources.
+
+### 1️⃣ First-class Azure Key Vault Secret support
+
+Aspire now supports `IAzureKeyVaultSecretReference`, a new primitive for modeling secrets directly in the app model. This replaces `BicepSecretOutputReference` and avoids creating a separate Key Vault per resource.
+
+You can now:
+
+- Add a shared Key Vault in C#
+- Configure services that support keys (e.g., Redis, Cosmos DB) to store their secrets there
+- Reference those secrets in your app as environment variables or via the Key Vault config provider
+
+Use KeyVault directly in your api:
+
+```csharp
+var builder = DistributedApplication.CreateBuilder(args);
+
+var vault = builder.AddAzureKeyVault("kv");
+
+var redis = builder.AddAzureRedis("redis")
+                   .WithAccessKeyAuthentication(vault);
+
+builder.AddProject<Projects.Api>("api").WithReference(vault);
+```
+
+Let the compute environment handle the secret management for you:
+
+```csharp
+var vault = builder.AddAzureKeyVault("kv");
+
+var redis = builder.AddAzureRedis("redis")
+                   .WithAccessKeyAuthentication(vault);
+
+builder.AddProject<Projects.Api>("api")
+       .WithReference(redis);
+```
+
+**Previous behavior:**
+
+  `azd` created and managed secrets using Bicep, with no visibility in the app model. Secrets were handled implicitly and couldn't be customized in C#.
+
+**New behavior in 9.2:**
+
+  Calling `WithKeyAccessAuthentication` or `WithPasswordAuthentication` now creates an actual `AzureKeyVaultResource` (or accepts a reference to one), and stores connection strings there. Secret names follow the pattern `connectionstrings--{resourcename}` to prevent naming conflicts with other vault entries.
+
+### 🔒 Improved default permissions for Azure Key Vault references
+
+When referencing a Key Vault, Aspire previously granted the broad **Key Vault Administrator** role by default. In 9.2, this has been changed to **Key Vault Secrets User**, which provides read-only access to secrets—suitable for most application scenarios.
+
+This update continues the security-focused improvements in this release.
+
+## 🚀 Deployment improvements
+
+We're excited to announce several new deployment features in .NET Aspire 9.2, including:
+
+### 📦 Built-in publishers (Preview)
+
+Publishing integrations (or simply "publishers") are a new extensibility point in .NET Aspire that allow you to define how your distributed application gets transformed into deployable assets. Rather than relying on an [intermediate manifest format](../deployment/manifest-format.md), publishers can now plug directly into the application model to generate Docker Compose files, Kubernetes manifests, Azure resources, or whatever else your environment needs.
```

Comment:
```suggestion
Publishers are a new extensibility point in .NET Aspire that allow you to define how your distributed application gets transformed into deployable assets. Rather than relying on an [intermediate manifest format](../deployment/manifest-format.md), publishers can now plug directly into the application model to generate Docker Compose files, Kubernetes manifests, Azure resources, or whatever else your environment needs.
```